src/api/source/base-windshaft.js

```diff
@@ -39,8 +39,11 @@ export default class BaseWindshaft extends Base {
     _generateURL(auth, config) {
```

Comment:
maybe we should rename url to urlS since this variable now contains an object with multiple urls inside